solvers/skglm.py

```diff
@@ -0,0 +1,37 @@
+from benchopt import BaseSolver
+from benchopt import safe_import_context
+
+with safe_import_context() as import_ctx:
+    from skglm.estimators import MultiTaskLasso
+
+
+class Solver(BaseSolver):
+    name = "skglm"
+    stopping_strategy = "iteration"
+
+    install_cmd = "conda"
+    requirements = ["skglm"]
+    references = [
+        'Q. Bertrand and Q. Klopfenstein and P.-A. Bannier and G. Gidel'
+        'and M. Massias'
+        '"Beyond L1: Faster and Better Sparse Models with skglm", '
+        'https://arxiv.org/abs/2204.07826'
```

Comment:
It's a Neurips 2022 paper now :)

Reply:
I added `NeurIPS 2022`. Is there an updated link to include?

Reply:
Nope, looks good.
Beware that strings are concatenated without spaces between them so you need `Gidel '`, `2022 "` etc